pkgs/development/python-modules/tensorflow-metadata/default.nix

```diff
@@ -0,0 +1,47 @@
+{ absl-py
+, buildPythonPackage
+, fetchFromGitHub
+, googleapis-common-protos
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "tensorflow-metadata";
+  version = "1.5.0";
+
+  src = fetchFromGitHub {
+    owner = "tensorflow";
+    repo = "metadata";
+    rev = "v${version}";
+    sha256 = "17p74k6rwswpmj7m16cw9hdam6b4m7v5bahirmc2l1kwfvrn4w33";
+  };
+
+  patches = [
+    ./build.patch
+  ];
+
+  # Default build pulls in Bazel + extra deps, given the actual build
+  # is literally three lines (see below) - replace it with custom build.
+  preBuild = ''
+    set -e
```

Comment:
Is `set -e` not the default in nix shell stuff already?

Reply:
Apparently yes (adding failing command aborts the build indeed even without this setting), thanks for noticing! Removed. Was confused by seeing it regularly in other derivations - looks like this is not a common knowledge?

Reply:
Yeah, that's a good question... I would certainly hope that `set -e` is the default but perhaps it hasn't always been that way? I have no idea...